management/cmd/management.go

```diff
@@ -86,5 +94,9 @@ var (
 func init() {
 	mgmtCmd.Flags().IntVar(&mgmtPort, "port", 33073, "server port to listen on")
 	mgmtCmd.Flags().StringVar(&mgmtDataDir, "datadir", "/var/lib/wiretrustee/", "server data directory location")
+	mgmtCmd.Flags().StringVar(&mgmtHostsConfig, "hosts-config", "/etc/wiretrustee/hosts-config.json", "Wiretrustee system hosts config (STUN, TURN, Signal, etc). These will be advertised to peers ")
```

Comment:
I wouldn't go with a config file only for hosts. A single config file with all the parameters makes more sense and follows more the standard of services we see, like MySQL, Apache, and Nginx. 

The tricky part is to balance what you have in the config file and the flags you are providing at runtime, which should take precedence over the configuration from the file. e.g., port and letsencrypt-domain. 

For hosts configuration, I understand the issue is the way we are providing them, but the way it is presented in the JSON file is not friendly and we could use the URL scheme to indicate TCP and UDP protocols. This would allow us to keep the scheme of the CLI when running the init command.

An improvement for making it more flexible to run in containers is to support configs provided by environment variables.